module2/homework/addEven/addEven.hpp

```diff
@@ -2,7 +2,17 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    int liczba; //liczba elementów w wektorze
+    int liczba3 = 0; //suma liczb zwracana przez funkcję
+    liczba = numbers.size();
+
+    for (int liczba2 = 0 && liczba3; liczba2<liczba; liczba2++) {
+
+        if (numbers[liczba2]%2==0) {
+
+            liczba3=numbers[liczba2]+liczba3;
+        }
+    }
+
```

Comment:
wrong format

Reply:
> wrong format
What do you mean by that?

Reply:
He meant bad code formatting :) I suggest something like that:
```suggestion
    for (int liczba2 = 0 && liczba3; liczba2 < liczba; liczba2++) {
        if (numbers[liczba2] % 2 == 0) {
            liczba3 = numbers[liczba2] + liczba3;
        }
    }
```

Variable names also should be more descriptive.